File: python/7 kyu/411.py
# https://www.codewars.com/kata/53d32bea2f2a21f666000256/train/python

# Passed

def largest(n, xs):
    sorted_list = sorted(xs)
    result = sorted_list[-n:] if n != 0 else []
    return result

output = largest(7, [9, 1, 50, 22, 3, 13, 2, 63, 5])
print(output)

File: nodejs/7 kyu/411.js
// https://www.codewars.com/kata/5a512f6a80eba857280000fc/train/javascript

// Passed

function nthSmallest(arr, pos) {
  let sortedArr = arr.sort((a, b) => a - b);
  result = sortedArr[pos - 1];
  return result;
}

const output = nthSmallest([15, 20, 7, 10, 4, 3], 3);
console.log(output);